### ISSUE TITLE
graphql-ruby 1.10.0 update broke the gem

### ISSUE THREAD

**Issue by Postmodum37** [exAspArk/graphql-guard#35]
```
undefined method `target' for AppSchema:Class
/.rvm/gems/ruby-2.6.5/gems/graphql-guard-1.3.0/lib/graphql/guard.rb:22:in `use'
/.rvm/gems/ruby-2.6.5/gems/graphql-1.10.0/lib/graphql/schema.rb:863:in `use'
```
https://github.com/rmosolgo/graphql-ruby/blob/master/CHANGELOG.md

**Comment by liang3404814** [comment]
Fix example for 1.10.0 compatibility: https://github.com/Shopify/graphql-batch/pull/116

**Comment by exAspArk** [comment]
@Postmodum37 thanks for opening the issue! @liang3404814 thanks for providing the example! :raised_hands:

I release a new version `1.3.1` with fixed compatibility.
